src/jc_voronoi.h

```diff
@@ -39,6 +39,9 @@ extern "C" {
     #define JCV_EDGE_INTERSECT_THRESHOLD 1.0e-10F
 #endif
 
+#ifndef JCV_EPSILON
+    #define JCV_EPSILON 1.0e-10F
```

Comment:
This is a _very_ small epsilon, which essentially translates to 0 for floats, which I guess wasn't your intention.

Also, I'd prefer the epsilon was defined in the same segment as the `JCV_REAL_TYPE` to make it clearer that it belongs to the type.

Reply:
Agreed, it's just a default value which the user could override if needed, depending on whether their use case calls for it.

Maybe we could do it this way, so the epsilon can be changed whether or not the user wants to change JCV_REAL_TYPE and also to make it clear that it belongs to the type:
````
#ifndef JCV_REAL_TYPE
    #define JCV_REAL_TYPE float
#endif

#ifndef JCV_REAL_TYPE_EPSILON
    #define JCV_EPSILON 1.0e-10F
#endif
````

Reply:
I think it's good to have it as a separate define as you suggested, it's just the default value that is very strange, as it doesn't really work with the default type (float).

Reply:
I agree, it is quite arbitrary, how about setting it to `FLT_EPSILON`?